### PR TITLE
Sailing app

### PR DESCRIPTION
**Pull request by fliuzzi02**
Hi, i made a simple app that connects via UDP to an OpenPlotter server which casts navigation data to all devices listening to a specific port on the network.

In my fork i also deleted the example app over which my app is based and added a small guide in the usage.md file

### CODE REVIEW

File: src/app/sailing/sailing_setup.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    lv_obj_t *sailing_foobar_switch_label = lv_label_create( sailing_foobar_switch_cont, NULL);
+    lv_obj_add_style( sailing_foobar_switch_label, LV_OBJ_PART_MAIN, &sailing_setup_style  );
+    lv_label_set_text( sailing_foobar_switch_label, "Always on display");
+    lv_obj_align( sailing_foobar_switch_label, sailing_foobar_switch_cont, LV_ALIGN_IN_LEFT_MID, 5, 0 );
+}
+
+static void sailing_foobar_switch_event_cb( lv_obj_t * obj, lv_event_t event ) {
+    switch( event ) {
+        case( LV_EVENT_VALUE_CHANGED ): Serial.printf( "switch value = %d\r\n", lv_switch_get_state( obj ) );
+                                        if( lv_switch_get_state( obj ) == 1 ) display_set_timeout( 300 );
```
Comment (sharandac) [inline src/app/sailing/sailing_setup.cpp:93]: These function need an review. Where is the display timeout reset to the origin after exiting/terminating?

Comment (fliuzzi02) [reply src/app/sailing/sailing_setup.cpp:93]: Forgot it, i'll add the function asap

Comment (sharandac) [reply src/app/sailing/sailing_setup.cpp:93]: a good and save way is [this](https://github.com/sharandac/My-TTGO-Watch/blob/master/CONTRIBUTING.md#applications) ...

File: src/app/sailing/sailing_main.cpp [hunk 2]
```diff
@@ -194,6 +194,7 @@ static void enter_sailing_setup_event_cb( lv_obj_t * obj, lv_event_t event ) {
 static void exit_sailing_main_event_cb( lv_obj_t * obj, lv_event_t event ) {
     switch( event ) {
         case( LV_EVENT_CLICKED ):       mainbar_jump_to_maintile( LV_ANIM_OFF );
+                                        display_set_timeout( 15 );
                                         break;
     }
 }
```
Comment (fliuzzi02) [inline src/app/sailing/sailing_main.cpp]: I did this because you don't necessarily want to set the timeout to max when entering the app

**Pull request merged**